src/forEachChild.ts

```diff
@@ -43,6 +43,11 @@ export function forEachChild(node: any, visitor: any): any {
     case SyntaxKind.CoverInitializedName:
       return visitNode(node.left, node, visitor) || visitNode(node.right, node, visitor);
     case SyntaxKind.BindingElement:
+      return (
+        visitNode(node.left, node, visitor) ||
+        visitNode(node.type, node, visitor) ||
+        visitNode(node.right, node, visitor)
```

Comment:
why using `||`? IMO, all the nodes need to be visited.

Reply:
Same as what Typescript does in their walker.